src/state/wallet/hooks.ts

```diff
@@ -184,8 +187,11 @@ export function useCurrencyBalances(
   )
 }
 
-export function useCurrencyBalance(currency?: Currency): CurrencyAmount<Currency> | undefined {
-  return useCurrencyBalances(useMemo(() => [currency], [currency]))[0]
+export function useCurrencyBalance(currency?: Currency, chainId?: ChainId): CurrencyAmount<Currency> | undefined {
+  return useCurrencyBalances(
+    useMemo(() => [currency], [currency]),
```

Comment:
nonsense

Reply:
```suggestion
    currency,
```

Reply:
old code sir, I dont touch it. just add custom chainId. haha